### PR TITLE
fastem: update of the GUI integration

### PR DESCRIPTION
**Pull request by Sabrina85**

### CODE REVIEW

**Comment by Sabrina85** [comment]
This pr is dependent on the acquisition manager implementation for fastem (pr in review).

File: src/odemis/gui/cont/streams.py [hunk 1]
```diff
@@ -3282,7 +3282,13 @@ def __init__(self, name, roc, colour, tab_data, project_panel, view_ctrl):
         self._project_panel = project_panel
         self._view_ctrl = view_ctrl
 
-        self.model = odemis.acq.fastem.FastEMROA(name, acqstream.UNDEFINED_ROI, roc)
+        self._asm = self._tab_data.main.asm
```
Comment (philipwinklerdelmic) [inline src/odemis/gui/cont/streams.py:3285]: Why do you make them (private) attributes? I don't think we need them anywhere else in this class, or do you plan on extending it?

**Review by philipwinklerdelmic — APPROVED** [review]

**Review by pieleric — APPROVED** [review]

**Comment by pieleric** [comment]
This PR has one tiny comment to fix, and some conflicts. Once handled, it should be fine to merge :-)

**Comment by Sabrina85** [comment]
Done. I just need to wrap up the acquisition manager first. Otherwise this code will not work.

**Pull request merged**